### ISSUE TITLE
showoff feature: Run zellij without installation, anywhere

### ISSUE THREAD

**Issue by raphCode** [zellij-org/zellij#1447]
Many users report they like zellij because they can use it on servers without installation.

Maybe we can go one step further and provide a small script that downloads a prebuilt zellij binary and runs it. Advantages:
- show off in comparison to tmux etc
- new users can try zellij with almost zero effort
- run zellij on remote machines without installation. We could even provide an ssh wrapper.

Based on complexity we can provide a oneliner to copy-paste or host something on zellij.dev and tell users to `curl zellij.dev/script | bash` (Safer alternative: save to file, review, then execute)

**Comment by tlinford** [comment]
On mac with m1, it prints `Unsupported cpu arch: arm64`.

**Comment by tlinford** [comment]
Got it running with:
```
#!/bin/bash

dir="/tmp/zellij/bootstrap"
mkdir -p "$dir"
cd "$dir" || exit 1

if [[ -x zellij ]]
then
    ./zellij "$@"
    exit 0
fi

case $(uname -m) in
    "x86_64"|"aarch64")
        arch=$(uname -m)
    ;;
    "arm64")
        arch="aarch64"
    ;;
    *)
        echo "Unsupported cpu arch: $(uname -m)"
        exit 1
    ;;
esac

case $(uname -s) in
    "Linux")
        sys="unknown-linux-musl"
    ;;
    "Darwin")
        sys="apple-darwin"
    ;;
    *)
        echo "Unsupported system: $(uname -s)"
        exit 1
    ;;
esac

url="https://github.com/zellij-org/zellij/releases/latest/download/zellij-$arch-$sys.tar.gz"
curl --location "$url" | tar -xz
./zellij "$@"

```

**Comment by raphCode** [comment]
I opened a PR here with an improved version of the script: https://github.com/zellij-org/zellij-org.github.io/pull/129

A minor inconvenience is that `curl zellij.dev/launch | bash` does not work:
```
failed to open terminal:
ENOTTY: Not a typewriter
```
This is because bash's stdin is connected to `curl` instead of the terminal and therefore it can't give zellij a pty.

The smallest I have come up with is:
`bash <(curl zellij.dev/launch)` (bash / zsh?)
`bash (curl zellij.dev/launch | psub)` (fish?)

All shells beside bash should be tested, I just looked at the fish manual, and for zsh I assumed bash compatibility.

I hope someone with web design skills can add the command to the main page more easily than me.
